website/content/en/status/report-2023-04-2023-06/capsicum-ktracing.adoc

```diff
@@ -0,0 +1,251 @@
+=== Security Sandboxing Using `ktrace(1)`
+
+Links: +
+link:https://github.com/jakesfreeland/freebsd-src/tree/ff/ktrace[ktrace branch] URL: https://github.com/jakesfreeland/freebsd-src/tree/ff/ktrace +
+
+Contact: Jake Freeland <jfree@FreeBSD.org>
+
```

Comment:
Here, I'd add an intro (a sentence or 2) explaining you're extending ktrace functionality to allow uses you describe below. Otherwise, readers who don't check history of the ktrace branch may be left wondering "why is this a status report and not an article?" as I was at first.

Reply:
This report was extracted from a handbook article that I'm writing on Capsicumizing programs. I figured that the excerpt explained the extensions to `ktrace` well, so I copied it over to a report. If the writing is too reminiscent of an article, I can cut it down.

Either way, I added a sentence explaining that `ktrace` now records `ECAPMODE` violations.